lib/internal/per_context/primordials.js

```diff
@@ -228,6 +228,11 @@ function copyPrototype(src, dest, prefix) {
   copyPrototype(original.prototype, primordials, `${name}Prototype`);
 });
 
+// Define Symbol.Disposed and Symbol.AsyncDispose
+// Until these are defined by the environment.
+primordials.SymbolDispose ??= primordials.SymbolFor('nodejs.dispose');
```

Comment:
@aduh95 the `node-core/prefer-primordials` lint rule does not seem to like this:

```
  128:3  error  Use `const { SymbolDispose } = primordials;` instead of the global       node-core/prefer-primordials
  129:3  error  Use `const { SymbolAsyncDispose } = primordials;` instead of the global  node-core/prefer-primordials
```
should the linting code be adjusted, or should this code be fixed?

Reply:
The lint rule doesn't take into account that the built-in object would ever be modified by the runtime, and I think it's a good thing that the linter complains about it. The error message could be improved, but you'd need to add an `eslint-disable` comment in any case.